scenes/game/game.gd

```diff
@@ -1026,20 +1026,21 @@ func _on_name_opponent_card_begin(event):
 	var player = event['event_player']
 	spawn_damage_popup("Naming Card", player)
 	var normal_only = event['event_type'] == Enums.EventType.EventType_ReadingNormal
+	var can_name_fake_card = event['event_type'] == Enums.EventType.EventType_Boost_NameCardOpponentDiscards
 	if player == Enums.PlayerId.PlayerId_Player:
 		# Show the boost window.
 		_on_opponent_reference_button_pressed()
 		selected_cards = []
 		select_card_require_min = 1
 		select_card_require_max = 1
-		var cancel_allowed = false
+		var cancel_allowed = can_name_fake_card
 		popout_instruction_info = {
 			"popout_type": CardPopoutType.CardPopoutType_ReferenceOpponent,
 			"instruction_text": "Name an opponent card.",
 			"ok_text": "OK",
-			"cancel_text": "",
+			"cancel_text": "Name Exodia",
```

Comment:
funny, but maybe just "Name Nothing - Reveal Hand" is better

Reply:
I guess you could leave it Exodia, but just add the Reveal Hand part to be clear what is going on for people who don't get it